compiler/plugins/target/ROCM/builtins/tuning/iree_default_tuning_spec_gfx942.mlir

```diff
@@ -2734,6 +2734,31 @@ transform.named_sequence
   transform.yield %matmul, %config : !transform.any_op, !transform.any_param
 }
 
+transform.named_sequence
+@match_matvec_4x16384x6656_f16_f16_f32(%matmul: !transform.any_op {transform.readonly})
+  -> (!transform.any_op, !transform.any_param) {
+  transform.iree.match.has_no_lowering_config %matmul : !transform.any_op
+
+  %mmt = transform.include @match_mmt_f16_f16_f32 failures(propagate) (%matmul)
+    : (!transform.any_op) -> !transform.any_op
+  %lhs = transform.get_operand %matmul[0] : (!transform.any_op) -> !transform.any_value
+  %rhs = transform.get_operand %matmul[1] : (!transform.any_op) -> !transform.any_value
+  transform.iree.match.cast_compatible_type %lhs = tensor<4x16384xf16> : !transform.any_value
+  transform.iree.match.cast_compatible_type %rhs = tensor<6656x16384xf16> : !transform.any_value
+  %config = transform.param.constant #iree_codegen.compilation_info<
+    lowering_config = #iree_gpu.lowering_config<{partial_reduction = [0, 0, 512], 
+                                                 subgroup_basis = [[1, 1, 1], [0, 1, 2]], 
+                                                 thread = [0, 0, 8], 
+                                                 thread_basis = [[1, 1, 64], [0, 1, 2]], 
```

Comment:
FYI, I renamed this to `lane_basis` here https://github.com/iree-org/iree/pull/21412